packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModalContainer.js

```diff
@@ -29,7 +34,7 @@ class CollectionsModalContainer extends Component {
     this.state = {
       active: false,
       newCollection: {
-        display_name: '',
+        displayName: '',
```

Comment:
This change is causing a couple of tests to fail. I think we might want to leave it as `display_name` despite the linter complaining. The resources' attributes are snake case.

Reply:
Oops! Looks like the linter automatically converted that one. I'll figure out how to make it ignore those errors and not automatically change this one.

Reply:
@srallen Fixed. I also told `standard` to ignore the lines that were throwing errors: https://github.com/zooniverse/front-end-monorepo/pull/661/files#diff-1277d75a3bea17c9e1e77ad4b9dc4dc3R40